src/mlx/jira_juggler/jira_juggler.py

```diff
@@ -593,9 +593,37 @@ def load_issues_from_jira(self, depend_on_preceding=False, sprint_field_name='',
         while busy:
             try:
                 issues = jirahandle.search_issues(self.query, maxResults=JIRA_PAGE_SIZE, startAt=self.issue_count,
-                                                  expand='changelog')
-            except JIRAError:
-                logging.error('Invalid Jira query "%s"', self.query)
+                                            expand='changelog')
```

Comment:
I assume the move of the file will mean that it needs some MANIFEST.in changes as well.

Reply:
`graft src` in MANIFEST.in handles the move fine.